code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -110,6 +110,13 @@
 	ammo_type = /obj/item/ammo_casing/shotgun/tranquilizer
 	materials = list(MAT_METAL=1750)
 
+/obj/item/ammo_box/tranqdarts

```

Comment:
Currently, all ammo boxes except shotgun shells, .38, .357, 7.62 stripper clip, and capslook like boxed and act like speed loaders. This would include 10mm, 9mm, .45 caliber, .45 caliber rubber, 40mm, 7.62mm boxes, foam force darts, and riot foam darts.